pkg/traceql/engine_metrics.go

```diff
@@ -20,6 +20,8 @@ const (
 	internalLabelBucket   = "__bucket"
 	maxExemplars          = 100
 	maxExemplarsPerBucket = 2
+	// NormalNaN is a quiet NaN. This is also math.NaN().
```

Comment:
I stole this  from prometheus: https://github.com/prometheus/prometheus/blob/849215d90cf48119bcc772603857fdc0b986ac4b/model/value/value.go#L22